wheel_of_random.py

```diff
@@ -0,0 +1,49 @@
+
+import random
+from random import randint
+import datetime
+
+
+def wheel_of_random():
+    person = []
+
+    data_n = open('Name.txt', 'r', encoding='utf-8')
+    name_r = random.choice(data_n.readlines()).split('\n')
+    person.append(''.join(name_r))
```

Comment:
```suggestion
    name_r = random.choice(data_n.read().split('\n'))
    person.append(name_r)
```
Не уверен на 100%, но мне кажется так будет проще.